### PR TITLE
76 outer mpgd layer geometry change 1

### PR DESCRIPTION
**Pull request by mposik1983**
### Briefly, what does this PR introduce?
Change outer MPGD layer to bar geometry and build using assembly.

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
Yes. Outer MPGD layer is not implemented into ACTS

### Does this PR change default behavior?
Yes. Outer MPGD layer is not implemented into ACTS

### CODE REVIEW

File: compact/tracker.xml [hunk 1]
```diff
@@ -133,16 +133,17 @@
   </documentation>
   <plugins>
     <plugin name="epic_FileLoader">
-      <arg value="cache:$DETECTOR_PATH:/opt/detector"/>
+      <arg value="cache:$DETECTOR_PATH"/>
```
Comment (wdconinc) [inline compact/tracker.xml:136]: Was there a reason for removing this? It is required to find the material maps in the container.

File: compact/tracker.xml [hunk 2]
```diff
@@ -133,16 +133,17 @@
...
-  <include ref="tracker/silicon_barrel.xml"/>
-  <include ref="tracker/mpgd_barrel.xml"/>
-  <include ref="tracker/silicon_disks.xml"/>
-  <include ref="tracker/support_service_assembly.xml"/>
+	     <include ref="tracker/vertex_barrel.xml"/>
+	     <include ref="tracker/silicon_barrel.xml"/>
+	     <include ref="tracker/mpgd_barrel.xml"/>
+	     <include ref="tracker/mpgd_dirc.xml"/>
+	     <include ref="tracker/silicon_disks.xml"/>
+	     <include ref="tracker/support_service_assembly.xml"/>
```
Comment (wdconinc) [inline compact/tracker.xml:147]: No tabs please :-)

File: compact/tracker/mpgd_dirc.xml [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+  <limits>
+  </limits>
+
+  <regions>
+  </regions>
+
+  <display>
+	  </display>
+
+	  <detectors>
```
Comment (wdconinc) [inline compact/tracker/mpgd_dirc.xml:63]: More tabs to be removed. Probably good to apply that generally.

File: src/MPGDDIRC_geo.cpp [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+      //  ------------------- MPGD foil 
+      //  --               -- Frame
+      //  --  gas volume   -- Frame
+      //  --               -- Frame
+      //  ------------------- MPGD foil
+
+      //Look for gas modules to subtract frame thickness from
+      //FIXME: these module names are hard coded for now. Should find 
+      //a way to set a arribut via the moduel tag to flag what components 
+      //need to have frame thickness subtracted.
```
Comment (wdconinc) [inline src/MPGDDIRC_geo.cpp:118]: Is this why this is implemented as a different geometry plugin? Mainly I'm hoping we can avoid code duplication between this and the tracker geometry plugin.

File: compact/tracker.xml (hunk 1 above)
Comment (mposik1983) [reply compact/tracker.xml:136]: Not sure why that is changed. I did not explicitly change it, oh maybe it was from sourcing my local build directory? Anyhow I think keeping it as is ($DETECTOR_PATH:/opt/detector ) is fine

File: src/MPGDDIRC_geo.cpp (hunk 4 above)
Comment (mposik1983) [reply src/MPGDDIRC_geo.cpp:118]: This is one reason. Another is that this is the only tracking code which uses rectangular geometry. The other right now I believe use  cylindrical geometries.

File: compact/tracker.xml (hunk 2 above)
Comment (mposik1983) [reply compact/tracker.xml:147]: Ok, let me remove the tabs and I will update

File: compact/tracker.xml (hunk 1 above)
Comment (mposik1983) [reply compact/tracker.xml]: Fixed

File: compact/tracker.xml (hunk 2 above)
Comment (mposik1983) [reply compact/tracker.xml]: fixed

File: compact/tracker/mpgd_dirc.xml (hunk 3 above)
Comment (mposik1983) [reply compact/tracker/mpgd_dirc.xml]: fixed

File: compact/tracker/mpgd_dirc.xml [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+  <regions>
+  </regions>
+
+  <display>
+  </display>
+
+   <detectors>
+   <detector id="MPGDDIRC_0_ID" name="MPGDDIRC_0" type="epic_MPGDDIRC" readout="MPGDDIRCHits" vis="TrackerVis">
+	    <dimensions width="MPGDDIRCModule_width" length="MPGDDIRCModule_length" height="MPGDDIRCModule_allowed_space" />
+	    <position x="0" y="0" z="MPGDDIRCModule_offset" />
```
Comment (wdconinc) [inline compact/tracker/mpgd_dirc.xml:66]: There are still tabs :-)

Comment (wdconinc) [reply compact/tracker/mpgd_dirc.xml:66]: `xmllint --format compact/tracker/mpgd_dirc.xml` or `sed -i `s/\t/  /g' compact/tracker/mpgd_dirc.xml` but some postprocessing would still be needed.

Comment (mposik1983) [reply compact/tracker/mpgd_dirc.xml]: Thanks. I think the tabs are gone now😬

**Pull request merged**